src/gbt.cpp

```diff
@@ -74,9 +54,9 @@ bool scanGBTPhasesLocal(localArgs *la, const uint32_t ohN, const uint32_t N, con
 
             // Check the VFAT status
             for (uint32_t vfatN = 0; vfatN < oh::VFATS_PER_OH; vfatN++) {
-                const bool linkGood = (readReg(la, stdsprintf("GEM_AMC.OH_LINKS.OH%hu.VFAT%hu.LINK_GOOD", ohN, vfatN)) == 1);
-                const bool syncErrCnt = (readReg(la, stdsprintf("GEM_AMC.OH_LINKS.OH%hu.VFAT%hu.SYNC_ERR_CNT", ohN, vfatN)) == 0);
-                const bool cfgRun = (readReg(la, stdsprintf("GEM_AMC.OH.OH%hu.GEB.VFAT%hu.CFG_RUN", ohN, vfatN)) != 0xdeaddead);
+                const bool linkGood = (readReg(stdsprintf("GEM_AMC.OH_LINKS.OH%hu.VFAT%hu.LINK_GOOD", ohN, vfatN)) == 1);
```

Comment:
also, i think we were going to bypass the `stdsprintf` in favour of other mechanisms of constructing strings, though perhaps the exact how was never settled (@lpetre-ulb, @mexanick, @lmoureaux)?

Reply:
Indeed, `stdsprintf` is `rpcsvc`-dependent and it would be better if completely removed from whole code base. The [exact how was never settle](https://github.com/cms-gem-daq-project/ctp7_modules/issues/141), but my proposal of using `std::stringstream` for logging and `std::string` for register names generation still stands.

Reply:
OK, good that was my preference as well